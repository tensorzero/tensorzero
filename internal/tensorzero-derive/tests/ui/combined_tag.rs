#[derive(tensorzero_derive::TensorZeroDeserialize)]
#[serde(tag = "tag", rename_all = "snake_case")]
enum Foo {
    Bar,
    Baz,
}

fn main() {}
