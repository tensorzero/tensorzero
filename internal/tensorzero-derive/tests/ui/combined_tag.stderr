error: TensorZeroDeserialize: Please split `#[serde(tag = "...")]` into its own attribute
 --> tests/ui/combined_tag.rs:2:1
  |
2 | #[serde(tag = "tag", rename_all = "snake_case")]
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
